.github/workflows/scorecard.yml

```
@@ -0,0 +1,42 @@
+name: Scorecard supply-chain security
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)